src/main/features/response/views/full-rejection/why-do-you-disagree.njk

```diff
@@ -0,0 +1,22 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, textArea, saveAndContinueButton, errorSummary %}
+
+{% set heading = 'Why do you disagree with the claim amount?' %}
```

Comment:
The same here, please story to remove duplicates with list of these we are introducing now. Mark that story with tech debt label.

Reply:
ROC-4282